#!/bin/bash

# Give path to your aligned reads (.bam files)
# cd /fastq (I am already in the path)

# Loop over all BAM files
for bam in *.bam; do
    start=$(date +%s)  # start time

    echo "Processing $bam ..."
    featureCounts -s 0 -a Homo_sapiens.GRCh38.114.gtf \
        -o quants/${bam%.bam}_featurecounts.txt \
        "$bam"

    end=$(date +%s)  # end time
    runtime=$(( (end - start) / 60 ))  # in minutes

    echo "âœ… Completed $bam in $runtime minutes."
    echo "------------------------------------"
done
